Doc/whatsnew/3.13.rst

```diff
@@ -2591,6 +2591,12 @@ Build Changes
   C extensions are now built with the :ref:`limited C API <limited-c-api>`.
   (Contributed by Victor Stinner in :gh:`85283`.)
 
+* Two new configure flags are added: :envvar:`CFLAGS_BUILTIN_MODULE` and
+  :envvar:`LDFLAGS_BUILTIN_MODULE`. These flags allows refistributors to
+  provide compile and linker flags that only apply to standard library extension
+  modules. This is particulary useful to provide options such as ``RUNPATH`` and
+  ``RPATH`` that need to be applied specifically to standard library modules due to
+  its different relative location. (Contributed by Pablo Galindo in :gh:`131842`.)
```

Comment:
You should add it at the bottom to a section similar to https://docs.python.org/dev/whatsnew/3.11.html#notable-changes-in-3-11-5